src/issues.rs

```diff
@@ -65,14 +65,14 @@ pub struct CreateResponse {
 }
 
 #[derive(Deserialize, Debug)]
-pub struct IssueResults {
+pub struct Paginated<T> {
     pub expand: String,
     #[serde(rename = "maxResults")]
     pub max_results: u64,
     #[serde(rename = "startAt")]
     pub start_at: u64,
     pub total: u64,
-    pub issues: Vec<Issue>,
+    pub values: Vec<T>,
```

Comment:
I didn't catch this as well but the name change would have broken deserialization as well :/